_data/tools.yml

```diff
@@ -1979,5 +1979,24 @@
   v2: false
   v3: true
   v3_1: true
+  
+  
+- name: Counterfact
+  category:
+    - parser
+    - sdk
+    - converters
+    - testing
+  link: https://goswagger.io/
```

Comment:
Copy and paste error.

Reply:
If you want to fix the link so it doesn't point to goswagger.io we can merge this.  The counterfact tool looks very useful.

Reply:
```suggestion
  link: https://counterfact.dev
```

Reply:
Thanks! I don't know why I didn't fix this when I saw it in October. I guess I was on the mobile app.